src/main/resources/log4j.properties

```diff
@@ -0,0 +1,7 @@
+log = .
+log4j.rootLogger = INFO, LAMBDA
+
+# Define the LAMBDA Appender
+log4j.appender.LAMBDA=com.amazonaws.services.lambda.runtime.log4j.LambdaAppender
```

Comment:
interesting, I didn't know that there was a magic appender for lambda logs. presumably it sends them to cloudwatch? or logs nicely to stdout to end up in cloudwatch?  why don't we send them to kinesis->kibana